src/gar.lua

```diff
@@ -173,7 +197,7 @@ function gar.decreaseOperatorStake(from, qty, currentTimestamp, msgId)
 	end
 
 	if gateway.status == "leaving" then
-		error("Gateway is leaving the network and withdraw more stake.")
+		error("Gateway is leaving the network and cannot withdraw more stake.")
```

Comment:
👌

Reply:
I found another error message that was incorrectly worded like this one in decreaseDelegateStake.  Fixed it!